### PR TITLE
Create "Get Started" page object in E2E typescript selenium tests

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Create "Get Started" page object in E2E typescript selenium tests

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/16208

### CODE REVIEW

File: tests/e2e/pageobjects/ide/GetStarted.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
+/*********************************************************************
```
Comment (rhopp) [inline tests/e2e/pageobjects/ide/GetStarted.ts]: Wouldn't better place for this pageobject be `/tests/e2e/pageobjects/dashboard/GetStarted.ts`?

Comment (Ohrimenko1988) [reply tests/e2e/pageobjects/ide/GetStarted.ts]: Definitely yes, thank you =)

**Review by rhopp — APPROVED** [review]

**Comment by dmytro-ndp** [comment]
@Ohrimenko1988: please, take a look at [Happy Path tests PR check failure](https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/blue/organizations/jenkins/Multiuser-Che-PR-check-E2E-Happy-path-tests-against-k8s/detail/Multiuser-Che-PR-check-E2E-Happy-path-tests-against-k8s/2271/pipeline):
```
ERROR: /tmp/e2e/pageobjects/ide/GetStarted.ts[19, 29]: 'CREATE_AND_OPEN_BUTTON_XPATH' is declared but its value is never read.

npm ERR! Linux 3.10.0-957.el7.x86_64
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "run" "lint"
npm ERR! node v8.10.0
npm ERR! npm  v3.5.2
npm ERR! code ELIFECYCLE
npm ERR! e2e@1.0.0 lint: `tslint --fix -p .`
npm ERR! Exit status 2
```

**Comment by Ohrimenko1988** [comment]
[ci-test]

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**